zulipterminal/ui_tools/boxes.py

```diff
@@ -491,6 +504,48 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     self.model.save_draft(message)
                 elif message != saved_draft:
                     self.view.controller.save_draft_confirmation_popup(message)
+
+        # Since, there are two keys mapped to GO_RIGHT
+        # i.e. ['right', 'l'], we dont need 'l' key to satisfy
+        # the condition: is_command_key("GO_RIGHT", key).
+        # FIXME: Refactor this condition?
+        elif key == 'right':
```

Comment:
Yes.
The `l` key would otherwise, instead of getting typed as a letter in `header_box`, behave exactly the same as `right` keypress(i.e. cycle focus).

Reply:
Great; can we make this behave correctly if other keys are used instead of those currently assigned? I'd prefer not to hard-code the assumption of `['right', 'l']` in the code.